src/docs/get-started/editor.md

```diff
@@ -74,21 +74,20 @@ To install these:
 
 ## Install VS Code
 
-VS Code is a lightweight editor with Flutter app execution and debug support.
+VS Code is a lightweight editor with Flutter app execution and debugs support.
 
 * [VS Code][], latest stable version
 
 ## Install the Flutter and Dart plugins
 
  1. Start VS Code.
- 1. Invoke **View > Command Palette...**.
+ 1. Invoke **View > Command Palette...**
```

Comment:
A list should be consistent, so either all should end in a period, or none. This is a short sentence, but it is a sentence.